packages/api/src/constants.js

```diff
@@ -1,13 +1,28 @@
-module.exports = {
-  BLOCK_TIME: 5000,
-  StateTransitionEnum: {
-    DATA_CONTRACT_CREATE: 0,
-    DOCUMENTS_BATCH: 1,
-    IDENTITY_CREATE: 2,
-    IDENTITY_TOP_UP: 3,
-    DATA_CONTRACT_UPDATE: 4,
-    IDENTITY_UPDATE: 5,
-    IDENTITY_CREDIT_WITHDRAWAL: 6,
-    IDENTITY_CREDIT_TRANSFER: 7
+const TenderdashRPC = require('./tenderdashRpc')
+
+class Constants {
+  genesisTime = new Date()
+
+  async init () {
+    this.genesisTime = (await TenderdashRPC.getGenesis()).genesis_time
+  }
+
+  static get StateTransitionEnum () {
+    return {
+      DATA_CONTRACT_CREATE: 0,
+      DOCUMENTS_BATCH: 1,
+      IDENTITY_CREATE: 2,
+      IDENTITY_TOP_UP: 3,
+      DATA_CONTRACT_UPDATE: 4,
+      IDENTITY_UPDATE: 5,
+      IDENTITY_CREDIT_WITHDRAWAL: 6,
+      IDENTITY_CREDIT_TRANSFER: 7
+    }
+  }
+
+  static get BLOCK_TIME () {
```

Comment:
Do we have to define everything is static? I think if we set in the `this` it will be still accessable via dot notation

Reply:
❌